documentation/Functions.md

```diff
@@ -159,8 +159,9 @@ path: The URL path on which is checked if the server is running
 ### npmInstall
 #### parameter 
 1. Path to the project where the dependencies from the package.json file are to be installed.
+2. (Optional) name of a package, global or local installation, array of npm arguments
```

Comment:
Here we should mention that the parameter are passed via a json object.
And that each of the parameter inside the json object is optional.